src/attic/pre-odk historic files/Makefile

```diff
@@ -0,0 +1,73 @@
+OBO= http://purl.obolibrary.org/obo
```

Comment:
do not use spaces in folder names! (`pre-odk historic file`), just `pre-odk is fine`